v0/destinations/mailchimp/transform.js

```diff
@@ -115,141 +161,169 @@ async function responseBuilderSimple(payload, message, mailChimpConfig) {
     },
     userId: message.userId ? message.userId : message.anonymousId
```

Comment:
lets refactor here to use `getfieldValueFromMessage`

Reply:
done